packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1871,10 +1907,7 @@ export class ContainerRuntime
 					}
 			}
 
-			// For back-compat, notify only about runtime messages for now.
-			if (runtimeMessage) {
-				this.emit("op", message, runtimeMessage);
-			}
+			this.emit("op", message, runtimeMessage);
```

Comment:
Technically, this is breaking change. I'd suggest hiding it behind same groupedBatchingEnabled() check, i.e. have ability to test it as a whole, and ability to fully disable all behavioral changes in case something goes wrong.